aspnet/tutorials/first-mvc-app/start-mvc.rst

```diff
@@ -1,17 +1,24 @@
-Getting started with ASP.NET Core MVC
+Getting started with ASP.NET MVC Core
 ==================================================
 
 By `Rick Anderson`_
 
-This tutorial will teach you the basics of building an ASP.NET Core MVC app using `Visual Studio`_.
+This tutorial will teach you the basics of building an ASP.NET MVC Core web app using `Visual Studio 2015 <http://go.microsoft.com/fwlink/?LinkId=532606>`__. 
 
-.. contents:: Sections:
-  :local:
-  :depth: 1
+Sections
+^^^^^^^^^^^^^^^^
+	- `Install Visual Studio and ASP.NET`_
+	- `Create a web app`_
 
-Create a web app
+Install Visual Studio and ASP.NET
 --------------------------------------
 
+Visual Studio is an IDE (integrated development environment) for building apps. Similar to using Microsoft Word to write documents, you'll use Visual Studio to create web apps. Install :doc:`ASP.NET Core and Visual Studio 2015 </getting-started/installing-on-windows>`
```

Comment:
@danroth27 This is holding up the big show. I need a fwlink for VS15.2 - 
our installer takes care of .Net CLI tools